src/Fantomas.Core.Tests/TestHelpers.fs

```diff
@@ -133,10 +133,12 @@ let tryFormatAST ast sourceCode config =
 
 let formatConfig = { FormatConfig.Default with StrictMode = true }
 
-let shouldNotChangeAfterFormat source =
+let checkFormat config source expected =
```

Comment:
Could you move this to `KeepMaxEmptyLinesTests`? It is only used there and I typically want to keep all the tests having the same structure.
This is ok to have, just make it clear that it is only used for `KeepMaxEmptyLines`.